packages/lib-classifier/src/translations/test/plugins.json

```diff
@@ -26,13 +27,19 @@
       "itsThis": "Translated"
     }
   },
+  "TextFromSubjectTask": {
+    "reset": "Translated"
+  },
+  "TextFromSubjectTasks": {
```

Comment:
I don't think this is used anywhere? I could be mistaken/misunderstanding, but I think the plural form of "TextFromSubjectTasks" can be deleted, and only a singular "TextFromSubjectTask" with related "reset" translation is needed.

Reply:
Good catch! I'll remove that key from all dictionaries before merging this PR, and make sure the correct key exists in Lokalise